index.html

```diff
@@ -40,31 +39,185 @@ <h1 class="heading_second">
         <ul class="social_links">
           <li>
             <a href="#">
-              <img src="./images/Vector-2.jpg" class=logo-icons alt=union> 
+              <img src="./images/Vector-2.jpg" class=logo-icons alt=union>
             </a>
           </li>
           <li>
             <a href="#">
-              <img src="./images/Linkedin-icon.jpg" class=logo-icons alt=union> 
+              <img src="./images/Linkedin-icon.jpg" class=logo-icons alt=union>
             </a>
           </li>
           <li>
             <a href="#">
-              <img src="./images/Vector-3.jpg" class=logo-icons alt=union> 
+              <img src="./images/Vector-3.jpg" class=logo-icons alt=union>
             </a>
           </li>
           <li>
             <a href="#">
-              <img src="./images/Vector-4.jpg" class=logo-icons alt=union> 
+              <img src="./images/Vector-4.jpg" class=logo-icons alt=union>
             </a>
           </li>
-          <li>  
+          <li>
             <a href="#">
               <img src="./images/Vector.jpg" class=logo-icons alt=union>
             </a>
           </li>
+        </ul>
+      </section>
+    </div>
+    <div class="card_container5">
+      <div class="work_card">
+        <div class="img_cardiv">
+          <img class="mobi" src="./images/secondimage.png" alt="icon" id="mobi-img" />
+        </div>
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post<br>
+            Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
           </ul>
+          <button class="btn1" type="button">See Project</button>
+        </div>
+      </div>
```

Comment:
- [ ] Please fix the responsive design for all the work cards, currently it doesn't stretch out to fill all the available space, you could achieve this removing all the fixed-width in your CSS and letting flex box manage the width by you.

![image](https://user-images.githubusercontent.com/52635378/164916624-d557ce34-5425-4964-acbb-c05d5b685d76.png)